components/camera/camera.go

```diff
@@ -171,6 +160,49 @@ func DecodeImageFromCamera(ctx context.Context, mimeType string, extra map[strin
 	return img, nil
 }
 
+// GetImageFromGetImages is a utility function to quickly implement GetImage from an already-implemented GetImages method.
+// It returns a byte slice and ImageMetadata, which is the same response signature as the Image method.
+// We use the mimeType arg to specify how to encode the bytes returned from GetImages, returning the first image in the response slice.
+func GetImageFromGetImages(ctx context.Context, mimeType string, cam Camera) ([]byte, ImageMetadata, error) {
+	images, _, err := cam.Images(ctx)
+	if err != nil {
+		return nil, ImageMetadata{}, fmt.Errorf("could not get images from camera: %w", err)
+	}
+	if len(images) == 0 {
+		return nil, ImageMetadata{}, errors.New("no images returned from camera")
+	}
+	imgBytes, err := rimage.EncodeImage(ctx, images[0].Image, mimeType)
```

Comment:
what does this do if timages[0].Image is not in the requested mime type, such as images[0].Image is a viam depth image but the user asked for a jpeg? Do you have a test for that case?

Reply:
It's difficult to safeguard against mimetype mismatches in `GetImageFromGetImages` because `GetImages` does not return mimetypes, and the image return type is an opaque `image.Image`. We rely on `rimage.EncodeImage` to unbox its type and handle appropriate behavior and errors.